bookstore/settings.py

```diff
@@ -50,6 +50,12 @@
 
     # own
     'products',
+
+    # bootstrap5
+    'bootstrap5',
+
+    # bootstrap-themes
+    'bootstrap_themes',
```

Comment:
Remember to `pip freeze > requirements.txt` next time you add any required Python packages.